doc/book/admin/upgrades/1.6-1.10.rst

```diff
@@ -0,0 +1,54 @@
+..  _admin-upgrades-1.6-1.10:
+
+Caveats when upgrading from Tarantool 1.6 to 1.10
+=================================================
+
+This page includes explanations and solutions to some common issues
+when upgrading the cluster from Tarantool 1.6 to 1.10.
+This will be of interest to anyone upgrading to the most recent Tarantool version,
+like 2.8.4, 2.10.0 and such, since taking an intermediate step with
```

Comment:
Let's replace 2.10.0 with 2.10.1, since it's more up to date as of the time of writing